http_post.go

```diff
@@ -128,10 +132,17 @@ func (n *HTTPPostNode) postRow(row *models.Row) {
 		return
 	}
 
+	// Set content type and other headers
 	req.Header.Set("Content-Type", "application/json")
 	for k, v := range n.c.Headers {
 		req.Header.Set(k, v)
 	}
+
+	// Set timeout
+	ctx, cancel := context.WithTimeout(req.Context(), h.timeout)
```

Comment:
Should this be `hn.Timeout`?

Reply:
that should have been n.timeout, not sure how I missed that, could have sworn it passed local tests...  pushed new change.